notebooks/crisp_anomaly_detection.ipynb

```diff
@@ -12,7 +12,7 @@
     "[![Open in Databricks](https://img.shields.io/badge/databricks-red?logo=databricks&style=for-the-badge)](https://www.databricks.com/try-databricks)\n",
     "[![View on GitHub](https://img.shields.io/badge/View%20on-GitHub-lightgrey?logo=github&style=for-the-badge)](https://github.com/gocrisp/analytics-blueprints-public/blob/main/notebooks/crisp_anomaly_detection.ipynb)\n",
     "\n",
-    "This notebook demonstrates how to perform naive outlier detection on sales and inventory data using z-scores. We will identify data points that are considered anomalies based on the criterion that their z-score exceeds |3.5| within time windows of sufficient size."
+    "Detecting anomalies in your data can help you identify unexpected patterns like stockouts, sales spikes, or pricing errors, so you can take action to improve inefficiencies, capitalize on gains, or address problems. This notebook demonstrates how to perform naive outlier detection on sales and inventory data using z-scores. We will identify data points that are considered anomalies based on the criterion that their z-score exceeds |3.5| within time windows of sufficient size."
```

Comment:
I tried adding a sentence here of some use cases for anomaly detection, but feel free to adjust if anything here doesn't feel accurate.